OpenAL32/alBuffer.cpp

```diff
@@ -20,10 +20,11 @@
 
 #include "config.h"
 
-#include <stdlib.h>
-#include <stdio.h>
-#include <assert.h>
-#include <limits.h>
+#include <cassert>
+#include <climits>
+#include <cstdio>
+#include <cstdlib>
```

Comment:
Won't these require using the `std::` namespace for the functions and types, e.g. `std::memcpy`?

Reply:
Yeah. Some implementation also have function in global namespace and std. I'll recheck usages. 

Reason for change is that `c` libs are less liked by templates.